src/osp/Active/activetypes.h

```diff
@@ -27,6 +27,11 @@ using UpdateOrderHandle = FunctionOrderHandle<void(void)>;
 using RenderOrder = FunctionOrder<void(ACompCamera const&)>;
 using RenderOrderHandle = FunctionOrderHandle<void(ACompCamera const&)>;
 
+struct ACompFloatingOrigin
+{
+    bool m_dummy;
```

Comment:
?

Reply:
Ideally there would be some sort of label or tag concept in EnTT that allows you to tag entities without actually requiring a stored component, but I'm not sure if such a thing exists and if not there's not much choice but to include a dummy member to make it a valid struct.

Reply:
EnTT supports tag structs just fine, just that it happened to not compile for this project specifically. I decided to work around it for in that case.

I looked throught the erorrs, and the compile issues are caused by the functions reg_emplace and reg_get in ActiveScene. They're short hands for calling emplace and get in its EnTT registry. Their return values are references, and changing it to decltype(auto), happens to fix it.